Makefile

```diff
@@ -235,3 +250,40 @@ regenerate-test-protos: ${PROTOC_GEN_SWIFTX}
 	for t in ${TEST_PROTOS}; do \
 		${PROTOC} --plugin=${PROTOC_GEN_SWIFTX} --swiftX_out=Tests/SwiftProtobufTests -I Protos Protos/$$t; \
 	done;
+
+
+# Helpers to put the Xcode project through all modes.
+
+# Grouping targets
+test-xcode: test-xcode-iOS test-xcode-macOS
```

Comment:
np.  not sure if you saw (I'm catching up on email), i backed this all out because i realized i broke clean builds.  Xcode was locally caching something.  I'll reland once I sort it all out.